Multisig/UI/Safe Management/Create Safe/Setup Safe/CreateSafeFormUIModel.swift

```diff
@@ -628,6 +628,7 @@ class CreateSafeFormUIModel {
     }
 
     func relaySubmit() {
+        Tracker.trackEvent(.relayUserExecTxPaymentRelay)
```

Comment:
proper place would be in didTapSubmit callback of the button as we track user action here.